bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/UserTarget.java

```diff
@@ -0,0 +1,70 @@
+package edu.gemini.spModel.target.env;
+
+import edu.gemini.shared.util.immutable.ImOption;
+import edu.gemini.shared.util.immutable.Option;
+import edu.gemini.spModel.target.SPTarget;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+/**
+ * Tuple of a user SPTarget and its type.
+ */
+public final class UserTarget implements Serializable {
```

Comment:
I'm not sure it matters since we don't try to maintain binary compatibility across releases.  If you do add a `serialVersionUID` but forget to update it after making an incompatible change the error is less clear.